composer.json

```diff
@@ -51,7 +51,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.8-dev"
+            "dev-master": "1.1-dev"
```

Comment:
Aaaargh, noooo, leaving 0.8-dev for 1.0 is wrong! Why you are in such a rush merging PRs :(